### PR TITLE
Send more consistent data in `update-groups` payload

### PR DESCRIPTION
**Pull request by Aberdeener**

### CODE REVIEW

**Comment by Derkades** [comment]
Not sure about, as far as I can see the API uses underscores for all fields? Only API methods use dashes

File: src/com/namelessmc/java_api/NamelessAPI.java [hunk 1]
```diff
@@ -97,11 +97,11 @@ public void sendMinecraftGroups(final int serverId, final Map<UUID, Set<String>>
 		groups.forEach((uuid, playerGroups) -> {
 			final JsonObject playerGroupsObject = new JsonObject();
 			playerGroupsObject.add("groups", gson.toJsonTree(playerGroups));
-			groupsJson.add(uuid.toString(), playerGroupsObject);
+			groupsJson.add(uuid.toString().replace('-', ''), playerGroupsObject);
```
Comment (Derkades) [inline src/com/namelessmc/java_api/NamelessAPI.java:100]: There's a special function `NamelessAPI.javaUuidToWebsiteUuid()`, but I totally forgot about it :P

**Comment by Aberdeener** [comment]
> I see, the serverInfo endpoint is the only endpoint that uses dashes! That's probably where you saw it. Maybe we should update the serverInfo endpoint instead? Just make the API send fields with underscores and with dashes for backwards compatibility.

Sounds good!

File: src/com/namelessmc/java_api/NamelessAPI.java (hunk 1 above)
Comment (Aberdeener) [reply src/com/namelessmc/java_api/NamelessAPI.java:100]: So used to single quotes 😢

**Comment by Derkades** [comment]
I was going to fix that myself, but now I don't have to, thanks!

**Pull request merged**